src/annotator/test/features-test.js

```diff
@@ -51,10 +56,10 @@ describe('features - annotation layer', function() {
     });
 
     it('should warn when accessing unknown flags', function() {
-      assert.notCalled(console.warn);
+      assert.notCalled(fakeWarnOnce);
```

Comment:
Given that the `sinon.stub` is re-initialized before each test, is this test useful?

Reply:
Good catch!